<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortareTopologicaV2Class</class>
 <widget class="QMainWindow" name="SortareTopologicaV2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortareTopologicaV2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="setOriented">
       <property name="text">
        <string>Oriented</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="checkCircuitsButton">
         <property name="text">
          <string>Check Circuits</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="adjacencyMatrixButton">
       <property name="text">
        <string>Adjacency Matrix</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="sortedButton">
       <property name="text">
        <string>Save Topological Sort</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="sortedGraphButton">
       <property name="text">
        <string>Show Sorted Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="rootButton">
         <property name="text">
          <string>Get Root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rootLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="treeCheckButton">
         <property name="text">
          <string>Is Tree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="isTreeLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="hasCircuitsLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="noCircuitsLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sortedLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sortaretopologicav2.qrc"/>
 </resources>
 <connections/>
</ui>
